jupyterhub/templates/hub/deployment.yaml

```diff
@@ -59,6 +59,9 @@ spec:
       imagePullSecrets:
         - name: hub-image-credentials
       {{- end }}  
+      {{- if .Values.hub.initContainers }}
+      {{- .Values.hub.initContainers | toYaml | trimSuffix "\n" | nindent 6 }}
```

Comment:
```suggestion
      initContainers:
        {{- .Values.hub.initContainers | toYaml | trimSuffix "\n" | nindent 8 }}
```

Reply:
The content of initContainers was added without the actual header before, this should fix that I think.